en/docs/security-announcements/cve-justifications/2025/BDSA-2022-1920.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "BDSA-2022-1920"
+category: security-announcements
+date: December 9 2025
+---
+
+# BDSA-2022-1920
+
+<p class="doc-info">WSO2 Products Impacted: No</p>
+<p class="doc-info">Customers Actions Required: No</p>
+---
+
+### REPORTED VULNERABILITY
+
+Apache PDFBox contains a bug that can cause an infinite loop when processing a PDF file. This could allow an attacker to supply a crafted PDF file and cause a denial-of-service (DoS). [^1][^2]
+
+### REPORTED PRODUCTS
+
+- WSO2 API Manager 4.6.0 and earlier
```

Comment:
Is this available in all the APIM versions?

Reply:
This justification was removed after an internal discussion. Hence, resolving the comment.